### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by AGraber** [IllidanS4/PawnPlus#26]
Since the plugin isn't really bound to players (or even SA-MP, given a proper compatible runtime that loads plugins as expected), it would be ideal to have some tests to ensure that everything is working properly and checking that nothing breaks with time.

I'll do some when I can, but I thought it would be ideal to have this written down somewhere.

**Comment by IllidanS4** [comment]
Definitely! I wanted to make a proper collection of tests before 1.0, but I never got to it.
